mautic/mautic
# Fixes broken installer

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | 
| BC breaks?    | 
| Deprecations? | 
| Fixed issues  |  

## Description
2.0 refactoring and new Doctrine dependencies introduced some new roadblocks for the installer which is broken currently.

This PR restores the installer.

## Steps to reproduce the bug (if applicable)
1. Delete the cache, delete app/config/local.php and go through installer.  It will fail installing the database fixtures.

## Steps to test this PR
1. Apply the PR, delete the cache, delete app/config/local.php and go through installer. It should be successful.
